end-to-end-tests/specs/results.spec.js

```diff
@@ -169,6 +169,10 @@ describe('Results Page', function() {
                 browser.elements('.//*[text()[contains(.,"OncoKB")]]').value[0].click();
                 browser.waitForExist("[class=onco-kb-0]", 10000);
 
+                // open PTM track
+                browser.elements('.//*[text()[contains(.,"PTM Sites")]]').value[0].click();
```

Comment:
browser.elements('*=PTM Sites')

Reply:
I just copied this logic from an existing test, I should've known there was something cleaner like this :)

Reply:
Actually, replacing `browser.elements('.//*[text()[contains(.,"PTM Sites")]]')` with `browser.elements('*=PTM Sites')` doesn't really work. Maybe this is the only way to select an option from `react-select` component?